cmd/tailscale/cli/file.go

```diff
@@ -324,7 +324,7 @@ func runFileGet(ctx context.Context, args []string) error {
 	for {
 		wfs, err = tailscale.WaitingFiles(ctx)
 		if err != nil {
-			return fmt.Errorf("getting WaitingFiles: %v", err)
```

Comment:
rather than remove this, use %w (wrapped error) rather than %v?

I assume you removed this just so `tailscale.IsAccessDenied` would work?

Reply:
that's right; i'm a bit new to golang, didn't know %w was a thing :+1: